clang/include/clang/Driver/Options.td

```diff
@@ -4188,6 +4188,10 @@ def fsycl_remove_unused_external_funcs : Flag<["-"], "fsycl-remove-unused-extern
   Group<sycl_Group>, HelpText<"Allow removal of unused `SYCL_EXTERNAL` functions (default)">;
 def fno_sycl_remove_unused_external_funcs : Flag<["-"], "fno-sycl-remove-unused-external-funcs">,
   Group<sycl_Group>, HelpText<"Prevent removal of unused `SYCL_EXTERNAL` functions">;
+def fsycl_allow_split_with_dependencies : Flag<["-"], "fsycl-allow-split-with-dependencies">,
+  Group<sycl_Group>, HelpText<"Allow dependencies between device code images">;
+def fno_sycl_allow_split_with_dependencies : Flag<["-"], "fno-sycl-allow-split-with-dependencies">,
```

Comment:
I'm not sure if we need approval from the options group for these, @mdtoguchi?

Reply:
@sarnex and @LU-JOHN, any new option (that we are inventing) that is meant to be visible should go through the options team workgroup.

Reply:
Add new option in separate PR.